db/config.c

```diff
@@ -690,6 +691,31 @@ static int read_lrl_option(struct dbenv *dbenv, char *line,
             dbenv->sibling_port[0][NET_REPLICATION] = port;
             dbenv->sibling_port[0][NET_SQL] = port;
         }
+    } else if (tokcmp(tok, ltok, "fdb_whitelist") == 0) {
+        /* expected parse line: 
+         * fdb_whitelist nodes node1 node2 ... databases db1 db2 ...
+         */
+        tok = segtok(line, len, &st, &ltok);
+#if 0
```

Comment:
Can we please get rid of commented-out code?

Reply:
Yep.